lib/commands/web.js

```diff
@@ -215,7 +215,7 @@ extensions.translateWebCoords = async function translateWebCoords (coords) {
   try {
     this.setImplicitWait(0);
     webview = await retryInterval(5, 100, async () => {
-      return await this.findNativeElementOrElements('-ios predicate string', `type = 'XCUIElementTypeWebView' AND visible = 1`, false);
+      return await this.findNativeElementOrElements('-ios predicate string', `type = 'XCUIElementTypeWebView'`, false);
```

Comment:
maybe it makes sense to change the locator to 

```
'-ios class chain', `**/XCUIElementTypeWebView[$visible = 1$]`
```

e.g. if a webview has at least one visible descendant element then